e2e/wdio/platform/tests/time-picker.e2e-spec.ts

```diff
@@ -0,0 +1,130 @@
+
+describe('Time picker suite', function() {
+    let timePickerPage: TimePickerPO;
+    timePickerPage = new TimePickerPO();
+    const {
+        activeTimePickerInput, timePickerButton, timePickerInput, timerExpanded,
+        activeTimePickerButton, errorBorder,
+        selectedHours, selectedMinutes, period,
+        navigationDownArrowButton, timeItem, setToNullButton, setValidTimeButton
+    } = timePickerPage;
+
+    beforeAll(() => {
+        timePickerPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForPresent(timePickerInput);
+    }, 1);
+
+    it('Verify in all the form factor user is able to see the date picker button and input field ', () => {
+        const buttonsCount = getElementArrayLength(timePickerButton);
+        const inputsCount = getElementArrayLength(timePickerInput);
+        expect(buttonsCount).toEqual(inputsCount);
+        for (let i = 1; i < buttonsCount; i++) {
+            waitForElDisplayed(timePickerButton, i);
+            waitForElDisplayed(timePickerInput, i);
+        }
+    });
+
+    it('Verify on click on the time picker button', () => {
+        const activeButtonsCount = getElementArrayLength(activeTimePickerButton);
+        for (let i = 1; i < activeButtonsCount; i++) {
+            sendKeys(['Escape']);
+            scrollIntoView(activeTimePickerButton, i);
+            click(activeTimePickerButton, i);
+            waitForElDisplayed(timerExpanded);
+            click(activeTimePickerButton, i);
+        }
+    });
+
+    it('Verify input field have placeholder', () => {
+        const inputsCount = getElementArrayLength(activeTimePickerInput);
+        for (let i = 0; i < inputsCount; i++) {
+            expect(['', null]).not.toContain(getAttributeByName(activeTimePickerInput, 'placeholder', i));
+        }
+    });
+
+    it('Verify on click on the input field ', () => {
+        const activeInputsCount = getElementArrayLength(activeTimePickerInput);
+        for (let i = 0; i < activeInputsCount; i++) {
+            sendKeys(['Escape']);
+            scrollIntoView(activeTimePickerInput, i);
+            setValue(activeTimePickerInput, text, i);
+            expect(getValue(activeTimePickerInput, i)).toBe(text);
+            expect(doesItExist(timerExpanded)).toBe(false);
+        }
+    });
+
+    it('Verify user is able to set time', () => {
+        const activeButtonsLength = getElementArrayLength(activeTimePickerButton);
+        for (let i = 0; i < activeButtonsLength; i++) {
+            scrollIntoView(activeTimePickerButton, i);
+            click(activeTimePickerButton, i);
+            selectHoursAndMinutes(11);
+            sendKeys(['Escape']);
+            expect(doesItExist(timerExpanded)).toBe(false);
+            expect(getValue(activeTimePickerInput, i)).toBe(time);
+        }
+    });
+
+    it('Verify null validity time picker', () => {
+        scrollIntoView(activeTimePickerButton, 4);
```

Comment:
why tested only for one example? 
Note this functionality on the second example leads to console error and brack the page.

Reply:
Reported the issue and added scenario (skipped for now)